actuator/src/main/java/org/tron/core/vm/program/Program.java

```diff
@@ -1083,12 +1193,25 @@ public DataWord getBalance(DataWord address) {
     return new DataWord(balance);
   }
 
+  public DataWord getRewardBalance(DataWord address) {
+    ContractService contractService = ContractService.getInstance();
+    long rewardBalance = contractService
+            .queryReward(TransactionTrace.convertToTronAddress(address.getLast20Bytes()), getContractState());
+    return new DataWord(rewardBalance);
+  }
+
   public DataWord isContract(DataWord address) {
     ContractCapsule contract = getContractState()
         .getContract(TransactionTrace.convertToTronAddress(address.getLast20Bytes()));
     return contract != null ? new DataWord(1) : new DataWord(0);
   }
 
+  public DataWord isSRCandidate(DataWord address) {
```

Comment:
To be consistent with `isContract`(ISCONTRACT) opcode, you should use the name/OpCode/Solidity function name  `isWitness`(ISWITNESS)`

- Contract/Witness/Normal is the type code of AccountType
- consistency across the whole TVM design really matters